docs/deployment/digital-ocean-apps.mdx

```diff
@@ -61,6 +61,8 @@ Navigate to [MongoDB Atlas](https://cloud.mongodb.com/), sign in to your account
 
 After cluster creation, you'll need to set up security. Select the authentication type (username and password) and create a user.
 
+P.S. - Please note that the generated password starts with a letter, if not, regenerate the password. (otherwise, DigitalOcean can incorrectly parse mongoURI)
```

Comment:
let's use [Callout Boxes](https://mintlify.com/docs/content/components/callouts)
I think, Warning Callout is more suitable here

```suggestion
<Warning>
Please be aware that the initial character of the generated password should be a letter. If it isn't, you'll need to create a new password. 
Failing to do this may lead to DigitalOcean parsing the `MONGO_URI` variable incorrectly.
</Warning>
```